docs/getting-started-guides/vagrant.md

```diff
@@ -5,10 +5,18 @@ Running kubernetes with Vagrant (and VirtualBox) is an easy way to run/test/deve
 ### Prerequisites
 1. Install latest version >= 1.6.2 of vagrant from http://www.vagrantup.com/downloads.html
 2. Install latest version of Virtual Box from https://www.virtualbox.org/wiki/Downloads
-4. Get or build a [binary release](binary_release.md)
 
 ### Setup
 
+Setting up a cluster is as simple as running:
+
+```
+export KUBERNETES_PROVIDER=vagrant
+curl -sS https://get.k8s.io | sh
```

Comment:
@derekwaynecarr The idea was to separate out developer guides from getting started stuff. GSG should be user focussed and have minimal distractions. If you like, I can clone up a Vagrant developer guide first.